packages/network/src/helpers/trackerHttpEndpoints.ts

```diff
@@ -49,17 +49,18 @@ const cachedJsonGet = (
 ): express.Application => {
     let cache: undefined | {
         timestamp: number
-        json: any
+        json: string
     }
     return app.get(endpoint, (req: express.Request, res: express.Response) => {
         staticLogger.debug('request to ' + endpoint)
         if ((cache === undefined) || (Date.now() > (cache.timestamp + maxAge))) {
             cache = {
-                json: jsonFactory(),
+                json: JSON.stringify(jsonFactory()),
                 timestamp: Date.now()
             }
         }
-        res.json(cache.json)
+        res.setHeader('Content-Type', 'application/json')
+        res.end(cache.json)
```

Comment:
Could use `res.send(cache.json)` to send the response as it will improve the generate the `ETag` header  (https://stackoverflow.com/a/49242271)